src/index.ts

```diff
@@ -70,11 +70,15 @@ function fetchToRequestResponse(opts: r.Options, res: f.Response) {
   const request = res.body as {} as r.Request;
   request.headers = opts.headers || {};
   request.href = res.url;
+  // headers need to be converted from a map to an obj
+  const headers = {} as Headers;
```

Comment:
`res.headers` has a `.raw()` method that returns the headers as an object expected by `request` users.

Reply:
See https://github.com/fhinkel/teeny-request/blob/master/src/index.ts#L185

Reply:
Cool!  But weird.  It looks like that method returns all header values wrapped in an array.  Are you seeing that?